[Tools] Convert source MSB3073: command exit  ...T4Generator.exe with code 3

### Current behavior

when run Run convert-sourcetree-to-winui.cmd it is report MSB3073: command exit  ...T4Generator.exe with code 3

```shell
       "C:\GitHub\uno\build\Uno.UI.Build.csproj" (target: RunAPISyncTool) (1) ->
       "C:\GitHub\uno\src\Uno.UI\Uno.UI.csproj" (target: GenerateMixins) (20:51) ->
       "C:\GitHub\uno\src\Uno.UI\MixinGeneration.targets" (target: _InnerGenerateMixins) (88:2) ->
       (target: _InnerGenerateMixins) ->
         C:\GitHub\uno\src\Uno.UI\MixinGeneration.targets(161,2): error MSB3073: command exit " "C:\GitHub\uno\src\Uno.UI\..\T4Generator\bin\Release\T4Generator.exe" C:\GitHub\uno\src\Uno.UI\Mixins\DependencyPropertyMixins.tt
        C:\GitHub\uno\src\Uno.UI\Mixins" with code 3.
```

### Expected behavior

do not fail

### How to reproduce it (as minimally and precisely as possible)

1. Run `x86 Native Tools Command Prompt for VS 2022 Preview`
2. Write `CD C:\GitHub\uno\build`
3. Run `convert-sourcetree-to-winui.cmd`

### Workaround

Before run `convert-sourcetree-to-winui.cmd` write `Platform=`

### Works on UWP/WinUI

_No response_

### Environment

Other

### NuGet package version(s)

_No response_

### Affected platforms

Build tasks

### IDE

Visual Studio 2022

### IDE version

17.1.0 Preview 5

### Relevant plugins

_No response_

### Anything else we need to know?

same problem with `x64 Native Tools Command Prompt for VS 2022 Preview`

--- a/build/convert-sourcetree-to-winui.cmd
+++ b/build/convert-sourcetree-to-winui.cmd
@@ -1,4 +1,11 @@
 @echo off
+setlocal ENABLEEXTENSIONS
+
+if '%Platform%' NEQ '' if '%CI%' NEQ '' goto errorci
+
+if '%Platform%' NEQ '' if '%CI%' == '' goto question
+
+:convert
 echo Make sure that :
 echo - crosstargeting_override.props is not defininig UnoTargetFrameworkOverride
 echo - This script is run from the build folder (not src/build)
@@ -9,3 +16,15 @@ dotnet run %myvar%\..
 popd
 msbuild Uno.UI.Build.csproj "/p:CombinedConfiguration=Release|AnyCPU;BUILD_BUILDNUMBER=test_test_8888" /m /t:RunAPISyncTool /clp:PerformanceSummary;Summary /bl
 pause
+END
+:errorci
+  echo Your system has the Platform environment variable to %Platform%, which is known to break some msbuild projects.
+  exit /B 1
+:question
+  echo Your system has the Platform environment variable to %Platform%, which is known to break some msbuild projects.
+  CHOICE /C YFC /M "Continue? Press Y for for continue with Platform=%Platform% , F for force Platform to empty or C for Cancel."
+  set result=%errorlevel%
+  If /I '%result%' == '2' (set Platform=)
+  If /I '%result%' == '3' (exit /B 1)
+  goto convert
+endlocal

